scripts/write_markdown_file.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs')
+const YAML = require('yaml')
+const { EOL } = require('os')
+
+module.exports = writeMarkdownFile = (filePath, frontMatter, body) => {
+  const frontMatterTerminator = '---'
+  const originStatement = '<!--- This file autogenerated from https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/scripts/generate_learning_path_markdown.js -->'
```

Comment:
Could update script name here in this comment.

Reply:
Will do. I will link to the folder, as there are 2 scripts that generate markdown now.